src/main/scala/io/iohk/ethereum/domain/Receipt.scala

```diff
@@ -0,0 +1,41 @@
+package io.iohk.ethereum.domain
+
+import akka.util.ByteString
+import io.iohk.ethereum.network.p2p.messages.PV63.TxLogEntryImplicits
+import io.iohk.ethereum.rlp.RLPImplicits._
+import io.iohk.ethereum.rlp.RLPImplicitConversions._
+import io.iohk.ethereum.rlp.{RLPDecoder, RLPEncodeable, RLPEncoder, RLPList}
+import org.spongycastle.util.encoders.Hex
+
+case class Receipt(
+                    postTransactionStateHash: ByteString,
+                    cumulativeGasUsed: BigInt,
+                    logsBloomFilter: ByteString,
+                    logs: Seq[TxLogEntry]
+                  ) {
+  override def toString: String = {
+    s"""
+       |Receipt{
+       |postTransactionStateHash: ${Hex.toHexString(postTransactionStateHash.toArray[Byte])}
+       |cumulativeGasUsed: $cumulativeGasUsed
+       |logsBloomFilter: ${Hex.toHexString(logsBloomFilter.toArray[Byte])}
+       |logs: $logs
+       |}
+       """.stripMargin
+  }
+}
+
+object Receipt {
```

Comment:
not sure if this should be part of domain, it is related to networking and communications, maybe something like
https://github.com/input-output-hk/etc-client/blob/phase/3/txExecution/src/main/scala/io/iohk/ethereum/network/p2p/messages/PV62.scala#L82